app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageItemComponents.kt

```diff
@@ -307,7 +312,12 @@ fun PreviewMessageSendFailureWarning() {
 @Composable
 fun PreviewMessageDecryptionFailure() {
     WireTheme {
-        MessageDecryptionFailure(mockHeader, MessageFlowStatus.Failure.Decryption(false)) { _, _ -> }
+        MessageDecryptionFailure(
+            mockHeader,
+            MessageFlowStatus.Failure.Decryption(false),
+            { _, _ -> },
+            null
+        )
     }
 }
```

Comment:
can we have a preview for when the protocol is `Proteus` and the decryption failure is shown?